### PR TITLE
agent.GetCurrentRoot returns local & published root

### LINKED ISSUE #194: IdentityAgent: return local & published root on agent.GetCurrentRoot()

### PR DESCRIPTION
**Pull request by arnaucube**
resolves #194

### CODE REVIEW

File: services/identityagentsrv/service.go [hunk 1]
```diff
@@ -374,8 +350,30 @@ func (a *Agent) ExportMT() ([][2]string, error) {
...
+		return nil, err
+	}
+
+	claim, err := core.NewClaimFromEntry(&merkletree.Entry{Data: *proofClaim.Leaf})
+	if err != nil {
+		return nil, fmt.Errorf("Error parsing proofClaim.leaf: %v", err)
+	}
+	claimSetRootKey, ok := claim.(*core.ClaimSetRootKey)
+	if !ok {
+		return nil, fmt.Errorf("Invalid claim type in form.proofAssignName.leaf")
```
Comment (ed255) [inline services/identityagentsrv/service.go]: The error message is not right

File: services/identityagentsrv/service.go [hunk 2]
```diff
@@ -374,8 +350,30 @@ func (a *Agent) ExportMT() ([][2]string, error) {
 	return mt, err
 }
 
+type CurrentRoot struct {
```
Comment (ed255) [inline services/identityagentsrv/service.go]: Maybe the fields need some json tags?

File: services/identityagentsrv/service.go (hunk 1 above)
Comment (arnaucube) [reply services/identityagentsrv/service.go]: oh, *despiste*, fixed

File: services/identityagentsrv/service.go (hunk 2 above)
Comment (arnaucube) [reply services/identityagentsrv/service.go]: done

**Comment by ed255** [comment]
:dog2:

**Pull request merged**